tests/configs/basic_app/k8s_cronjobs.yml

```
@@ -0,0 +1,23 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  annotations: {app.kubernetes.io/managed-by: kubetools}
+  labels: {kubetools/name: generic-cronjob, kubetools/project_name: generic-app,
+    kubetools/role: cronjob}
+  name: generic-cronjob
+spec:
+  schedule: "*/1 * * * *"
+  concurrency_policy: "Replace"
+  suspend: false
+  jobTemplate:
+    spec:
+      template:
+        labels: {kubetools/name: generic-cronjob, kubetools/project_name: generic-app,
+          kubetools/role: cronjob}
```

```suggestion
        labels: {
          kubetools/name: generic-cronjob,
          kubetools/project_name: generic-app,
          kubetools/role: cronjob
        }
```
Same format suggestion if it works

---

Done

---

7f3c135a16930c0508371fab4c19197f91ebd507